pydis_site/apps/content/templatetags/page_github.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from django import template
+
+register = template.Library()
+
+
+@register.filter()
+def page_github(current_uri: str) -> str:
+    """
+    Return the corresponding GitHub page for the current site page.
+
+    Return a directory if the page is a directory on GitHub otherwise return the Markdown file
+    """
+    github_uri = current_uri.replace(
+        "http://pythondiscord.local:8000/pages/",
+        "https://github.com/python-discord/site/tree/main/pydis_site/apps/content/resources/"
+    )
+    github_uri = github_uri[:-1] if github_uri.endswith("/") else github_uri
```

Comment:
I'm not sure why you add the trailing slash just to remove it in the next line, seems like you could have just not have the trailing line in the replacement string.